README.md

```diff
@@ -15,7 +15,7 @@ The [Causal Labs](https://www.causallabs.io) iOS SDK integrates Causal with nati
 ### [CocoaPods](http://cocoapods.org)
 
 ```ruby
-pod 'CausalLabsSDK', '~> 0.3.0'
```

Comment:
Why isn't the old one 0.4.0

Reply:
This was something that was missed with the last release.  I'm going to update the release instructions now to ensure that this is updated along with the podspec file.

Reply:
PR with related change can be found [here](https://github.com/CausalLabs/causal/pull/1177)